apps/opt.c

```diff
@@ -326,6 +326,30 @@ int opt_int(const char *value, int *result)
     return 1;
 }
 
+static STRSTR_PAIR base[] = {
+    {"0x", "hex"},
+    {"0X", "hex"},
+    {"0", "oct"},
+    {NULL}
+};
+
+#define opt_number_error(v, b) \
```

Comment:
It effectively says that *if* it's a macro [that can't be written as statement], then function-*looking* one is preferred. It's does not say that function-looking macro is preferred over function. Choice between macro and function is not part to style guide. Normally macros (or inline function) is about performance, but this code is as far from critical as moon. In other words there is no reason for why this is not a static function.